js/battle.js

```diff
@@ -3219,7 +3219,16 @@ var Battle = (function () {
 			if (kwargs.prepare || kwargs.anim === 'prepare') {
 				this.prepareMove(pokemon, move, target);
 			} else if (!kwargs.notarget) {
-				(kwargs.anim ? Tools.getMove(kwargs.anim) : move).anim(this, [pokemon.sprite, target.sprite]);
+				move = kwargs.anim ? Tools.getMove(kwargs.anim) : move;
```

Comment:
`getMove` is unavailable (or, to be exact, provides no useful information) in replays.

We should pass `[spread]` or something from the server for this, instead of reading `move.target`.